.github/workflows/scan.yml

```
@@ -0,0 +1,29 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  schedule:
+    # Daily
+    - cron: "23 7 * * *"
+  push:
+    paths:
+      - .github/workflows/*
+  pull_request:
+    paths:
+      - .github/workflows/*
```

Let's make this future-proof if the project ever starts using in-tree actions. Also, I'd still run it on PR merge every time, for faster feedback:
```suggestion
  pull_request:
    paths:
      - .github/actions/*/action.yml
      - .github/workflows/*.yml
```